spec/lib/parameter_substitution/expression_spec.rb

```diff
@@ -58,6 +58,10 @@ def parse_expression(parameter_expression, context_options = {})
       it "convert raw output if there is more than one parameter" do
         expect("red1").to eq(parse_expression("<color><integer>", destination_encoding: :raw).evaluate)
       end
+
+      it "handles nested methods" do
```

Comment:
I did not find any test cases that specifically cover nesting method calls. If I missed it, this test could be removed.

Reply:
Those are all up in the parser because it's part of the parser's job to split this apart:
https://github.com/Invoca/parameter_substitution/blob/master/spec/lib/parameter_substitution/parser_spec.rb#L225-L285